packages/components/src/components/atoms/form/TextFieldTagsMultiple.tsx

```diff
@@ -0,0 +1,180 @@
+import { styled } from '@mui/system'
+import { FormHelperText, InputLabel } from '@mui/material'
+import IonIcon from '../IonIcon/IonIcon'
+import InfoTooltip from './InfoTooltip'
+import FormControl from './FormControl'
+import TextFieldTags from './TextFieldTags'
+import Button from '../buttons/Button'
+
+import {
+  IOptionsTags,
+  ITransformedLimitations,
+} from '@elastic-suite/gally-admin-shared'
+import DropDown from './DropDown'
+
+const CustomMultipleTextFieldsTags = styled('div')(({ theme }) => ({
+  display: 'flex',
+  gap: theme.spacing(2.5),
+  flexDirection: 'column',
+}))
+
+const CustomCloseTagsByOperator = styled('div')(({ theme }) => ({
+  position: 'absolute',
+  right: '-5px',
+  top: '-5px',
+  borderRadius: '50%',
+  background: theme.palette.colors.neutral[600],
+  color: 'white',
+  display: 'flex',
+  padding: theme.spacing(0.5),
+  zIndex: '9',
+  cursor: 'pointer',
+}))
+
+const CustomSelectOperator = styled('div')(({ theme }) => ({
+  display: 'flex',
+  gap: theme.spacing(1),
+  alignItems: 'center',
+  flexDirection: 'row',
+}))
+
+export interface ITextFIeldTagsForm {
+  disabled?: boolean
+  error?: boolean
+  fullWidth?: boolean
+  infoTooltip?: string
+  helperText?: ReactNode
+  helperIcon?: string
+  label?: string
+  margin?: 'none' | 'dense' | 'normal'
+  required?: boolean
+  options: IOptionsTags[]
+}
+
+export interface ITextFieldTag extends ITextFIeldTagsForm {
+  value: ITransformedLimitations
+  onChange: (operator: string | string[], data?: string[] | string) => void
+}
+
+function TextFieldTagsMultiple(props: ITextFieldTag): JSX.Element {
+  const {
+    value,
+    onChange,
+    disabled,
+    required,
+    error,
+    fullWidth,
+    helperText,
+    helperIcon,
+    label,
+    margin,
+    infoTooltip,
+    options,
+  } = props
+
+  const optionsListAvailable: (IOptionsTags & { disabled?: boolean })[] =
+    options.map((item) => {
+      if (value[item.value]) {
+        return { ...item, disabled: true }
+      }
+      return item
+    })
+
+  const [optionDefault, setOptionDefault] = useState<undefined | IOptionsTags>()
+
+  useEffect(() => {
+    setOptionDefault(
+      optionsListAvailable.find(
+        (item) => optionDefault?.value === item.value && !item?.disabled
+      ) ?? optionsListAvailable.find((item) => !item?.disabled)
+    )
+  }, [optionsListAvailable, optionDefault])
+
+  return (
+    <FormControl error={error} fullWidth={fullWidth} margin={margin}>
+      {Boolean(label || infoTooltip) && (
+        <div style={{ marginBottom: '24px' }}>
+          <InputLabel shrink required={required}>
+            {label}
+            {infoTooltip ? <InfoTooltip title={infoTooltip} /> : null}
+          </InputLabel>
+        </div>
+      )}
+      <CustomMultipleTextFieldsTags>
+        {disabled ? (
+          <TextFieldTags disabledValue="disabled" disabled />
+        ) : (
+          Object.entries(value).map((item) => {
```

Comment:
Si tu utilises `Object.entries`, destructure tout de suite ce que tu appelle `item` pour directement récupérer la clé et la valeur:
```suggestion
          Object.entries(value).map(([key, value]) => {
```